### PR TITLE
Create projects.dat file if not existing earlier

### PR DESCRIPTION
**Pull request by Coder-Manan**
Solves #3

### CODE REVIEW

File: Projects_To_Do_List.py [hunk 1]
```diff
@@ -2,7 +2,13 @@ def Projects_To_Do_List():
     Running = True
     while Running:
         import pickle
-        Projects = pickle.load(open("Projects_To_Do_List/Projects.dat","rb"))
+        try:
```
Comment (Veerhan-glitch) [inline Projects_To_Do_List.py:5]: hey everything looks good but can you pls explain me why did you use try-except instead of wb+?

Comment (Coder-Manan) [reply Projects_To_Do_List.py:5]: No specific reason.

**Comment by Coder-Manan** [comment]
The PR is being shown as `Not Accepted` on the Hacktoberfest site. Pls look into it @Veerhan-glitch

**Comment by Veerhan-glitch** [comment]
hey @Coder-Manan pls check and conform if this and all other of ur pr are being shown as Accepted

**Comment by Coder-Manan** [comment]
Yes @Veerhan-glitch it is now showing the requests as accepted. Thanks!

**Comment by Veerhan-glitch** [comment]
Np, just keep on making good and meaningful contributions to the community.
Happy coding.

**Pull request merged**